rhc-ose-ansible/roles/storage/README.md

```diff
@@ -0,0 +1,59 @@
+# storage Ansible Role
+
+Configures Disk partitions, VGs, LVs, filesystem, and mount points
```

Comment:
The current implementation doesn't seem to support more than just one partition, which may be fine, but personally I'd like to see support for handling more partitions (both primary and extended). However, if you choose to keep it to just one partition for now, it should be noted in the README as a highlighted "note:" line as the line above says "partitions" and hence may not be quite accurate.